pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -145,8 +145,10 @@ ansiColor('xterm') {
         parallel(branches)
       }
 
-      stage('[AWS] Update ssm') {
-        folioHashCommitCheck.updateBuildSha(currentBuildSha)
+      if (currentBuildSha) {
```

Comment:
In case of **currentBuildSha** equals to null or empty
You'll not have **stage('[AWS] Update ssm')**.
The number of stages and their order is changed.
Due to this fact we loose visual history of Pipeline runs where
**stage('[AWS] Update ssm')** still exist.

Reply:
Fixed